tests/emscripten_api_browser_infloop.cpp

```diff
@@ -19,8 +20,8 @@ struct Class {
     printf("waka %d\n", x++);
 
     if (x == 7 || x < 0) {
-      REPORT_RESULT(x);
       emscripten_cancel_main_loop();
+      exit(x);
```

Comment:
Including this, there are several places the order of `emscripten_cancel_main_loop` and `exit` has changed.. Why is that?